sloppy-deployment/entrypoint.sh

```diff
@@ -1,6 +1,16 @@
 #!/bin/bash -eu
 
+# Name of the Sloppy YAML configuration file
 SLOPPY_OUTPUT_FILE="sloppy.yml"
+# Name of the directory (which should be host volume) to where a copy of
+# $SLOPPY_OUTPUT_FILE is saved.
+SLOPPY_SAVE_OUTPUT_DIR="${SLOPPY_SAVE_OUTPUT_DIR:-}"
```

Comment:
Either use `SLOPPY_SAVE_OUTPUT` as described in the `README.md` or change `SLOPPY_SAVE_OUTPUT` in the docs to `SLOPPY_SAVE_OUTPUT_DIR`.  😉

Reply:
Oh my, I hope it's ok now.